handbook/repository/system-e2e.md

```diff
@@ -8,32 +8,44 @@ When testing an app/project you need to first start the app, then test it with P
 
 ## ⚡ TL;DR
 
-- Start the application: `yarn dev/init <app> && yarn dev <app>`
-- Test the app: `yarn playwright menu --e2e --project <path/to/your/app>`
+- Start the application: `yarn dev-init <app> && yarn dev <app>`
+- Test the app: `yarn playwright system-e2e/.*/<name-of-your-app>`
```

Comment:
I think it might be useful to explain the thinking behind this filter argument since it confused me quite a bit reading these docs (regexp in my shell!?).

Basically how it is regexp matching against the "path" of all *.spec.ts files in the entire monorepo. 

Which is why all of these work:

- `app/system-e2e` or even just `system-e2e`
- `service-portal/acceptance`
- `'system-e2e/.*/service-portal`

But these don't work:

- `service-portal` since it matches other non-playwright test files.
- `'system-e2e/*/service-portal'` because it's RegExp, not glob pattern.